### DISCUSSION TITLE
Synchronous events processing

### DISCUSSION THREAD

**Discussion by pchar2** [miguelgrinberg/python-socketio#1121] [Q&A]
Hello,

Is there a way to create a client which will handle events using a single thread ?

I have a use case where I need each events to be processed sequentially.

**Comment by miguelgrinberg** [comment]
Not right now, unfortunately. I have the idea to create a synchronous interface on top of the asynchronous client (and possibly the server as well), but I haven't done that yet.

**Reply by pchar2** [reply]
Thanks for the quick answer.

I will deal with event creation time for the moment.

Have a nice day
